src/ChatAvatars/props.tsx

```diff
@@ -0,0 +1,6 @@
+import { ChatProps, PersonProps } from '../util/interfaces';
+
+export interface Props {
```

Comment:
Nothing to do here but something for discussion: Should we be stripping props down to the bare minimum requirements or is an entire chat object just fine to pass in?

Reply:
yeah great point- I'm gonna change it to just pass in `people` from the chat